tests/st/calicoctl/test_default_pools.py

```diff
@@ -0,0 +1,180 @@
+# Copyright (c) 2015-2016 Tigera, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import netaddr
+import yaml
+from nose_parameterized import parameterized
+
+from tests.st.test_base import TestBase
+from tests.st.utils.docker_host import DockerHost
+from tests.st.utils.exceptions import CommandExecError
+from tests.st.utils.utils import ETCD_CA, ETCD_CERT, \
+    ETCD_KEY, ETCD_HOSTNAME_SSL, ETCD_SCHEME, get_ip, wipe_etcd, retry_until_success
+
+# Load the calico-node we've just built
+POST_DOCKER_COMMANDS = ["docker load -i /code/calico-node.tar"]
+if ETCD_SCHEME == "https":
```

Comment:
These ADDITIONAL_DOCKER_OPTIONS are now defined in docker_host.py - better to use those.